cime_config/machines/config_machines.xml

```diff
@@ -3532,21 +3532,132 @@
         <command name="load">essl/6.3.0</command>
         <command name="load">netlib-lapack/3.8.0</command>
       </modules>
-      <modules compiler="pgi.*">
-        <command name="load">nvhpc/21.11</command>
-      </modules>
       <modules compiler="pgigpu">
+        <command name="load">nvhpc/21.11</command>
         <command name="load">cuda/10.1.243</command>
       </modules>
       <modules compiler="gnugpu">
-	   <command name="load">cuda/10.1.168</command>
-      </modules>
-      <modules compiler="ibm.*">
-        <command name="load">xl/16.1.1-10</command>
+        <command name="load">gcc/9.3.0</command>
```

Comment:
Hmm...this is going to revert the GCC+CUDA modules that are currently working on GPUs. I think you need a new rebase from current master.